app/templates/courses/activate-licenses-modal.jade

```diff
@@ -8,7 +8,8 @@ block modal-header-content
 
 block modal-body-content
   - var numToEnroll = state.get('visibleSelectedUsers').length
-  - var unusedEnrollments = view.prepaids.totalMaxRedeemers() - view.prepaids.totalRedeemers()
+  - var unusedEnrollments = view.prepaidByGroup[view.selectedPrepaidType]
+  -// view.prepaids.totalMaxRedeemers() - view.prepaids.totalRedeemers()
```

Comment:
Don't think we need this comment line around, right?

Reply:
yeah, comment will be removed before merge